src/connectors/jqueryConnector.js

```diff
@@ -0,0 +1,17 @@
+export default ($) => (sdk) => sdk(
+    (url, data, headers) => new Promise((resolve, reject) => $.ajax(url, {
+        method: 'POST',
+        data,
+        dataType: 'json',
+        headers,
+        success: resolve,
+        error: reject
+    })),
+    (url, data, headers) => new Promise((resolve, reject) => $.ajax(url, {
+        method: 'GET',
+        data: JSON.stringify(data),
```

Comment:
Does jquery not take care of query string Parameters or is that what it uses data for here?